core/storage/feedback/gae_models.py

```diff
@@ -256,11 +362,11 @@ class SuggestionModel(base_models.BaseModel):
     author_id = ndb.StringProperty(required=True, indexed=True)
     # ID of the corresponding exploration.
     exploration_id = ndb.StringProperty(required=True, indexed=True)
-    # The exploration version for which the suggestion was made.
+    # The exploration version for which the suggestions were made.
```

Comment:
No -- singular. Each instance of this model represents one suggestion. Ditto below.